setup.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+
+read -p "Project name: " PROJECT_NAME
+
+echo "[1/4] >> Creating a new project based on naveteam/flutter-boilerplate."
+git clone -b main https://github.com/naveteam/flutter-boilerplate $PROJECT_NAME
+
+echo "[2/4] >> Initializing a new git repository."
+cd $PROJECT_NAME
+rm -rf .git
+git init --initial-branch=main
+
+echo $(cat .env.example) >> .env
```

Comment:
Aqui não caberia melhor ao invés de um `cat` e direcionar isso pra um arquivo colocar um `cp` direto?
```suggestion
cp .env.example .env
```